db/models.py

```diff
@@ -1 +1,32 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
```

Comment:
The `bonus` field in the `Skill` model is defined as a `CharField`. If `bonus` is intended to store numerical values (e.g., a bonus point or percentage), consider using a numerical field type like `IntegerField` or `FloatField` instead. This will ensure data integrity and allow for numerical operations if needed.